packages/integration-tests/api/api.js

```diff
@@ -104,7 +104,8 @@ async function callCumulusApi({ prefix, payload: userPayload, userParams = {} })
 function verifyCumulusApiResponse(response, acceptedCodes = []) {
   const successCodes = [200].concat(acceptedCodes);
   if (!successCodes.includes(response.statusCode)) {
-    throw new Error(response.body);
+    const errorText = response.body ? response.body : response.errorMessage;
```

Comment:
Are we sure that `response.errorMessage` will exist if `response.body` does not?

Reply:
Not fully sure.   It just did in at least two instances.  If *both* do not exist, it will return undef like it does now....